src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import Logo from "./Logo";
+import Nav from "./Nav";
+import { getCurrentPageForUrl, getUpdateMenu } from "../utils";
+import headerData from "../headerData.json";
+
+const Header = () => {
+  const { pathname } = window.location;
+  const { nav } = headerData;
+  const updatedNav = getUpdateMenu(pathname, nav);
```

Comment:
Que es lo que actualiza? le podrías poner un nombre más descriptivo?

Reply:
Ahh, estás siguiendo los nombres originales de la webapp, tal vez convenga dejarlo así entonces.

Tal vez por ahora se puede poner un comentario cómo:

```suggestion
  # Recursively set isSelected property in menu items
  const updatedNav = getUpdateMenu(pathname, nav);
```

Y en un futuro podemos renombrar y refactorizar.

Reply:
Listo. Agregado para la proxima PR.